database/Models/index.js

```diff
@@ -1,13 +1,14 @@
 const { Sequelize } = require('sequelize');
 const { applyAdditionalSetup } = require('./additional-setup');
+const dotenv = require('dotenv');
```

Comment:
Is this needed to get reference to your env variables?

Reply:
Yes, there are other ways.. but the `dotenv` package makes it very straightforward and configurable